babel.config.js

```diff
@@ -1,4 +1,8 @@
-const plugins = ['@babel/plugin-proposal-class-properties', '@babel/plugin-proposal-object-rest-spread'];
+const plugins = [
+  '@babel/plugin-proposal-class-properties',
+  '@babel/plugin-proposal-object-rest-spread',
```

Comment:
We might be able to remove that depending on the Electron version we use

Reply:
https://github.com/wireapp/wire-desktop/pull/3330/files